rewrite-weblogic/documentation/content/Migrate to WebLogic 151100.md

```diff
@@ -0,0 +1,86 @@
+# Migrate to WebLogic 15.1.1
+**com.oracle.weblogic.rewrite.UpgradeTo1511**
+
+This recipe applies the changes required for migrating applications to WebLogic Server 15.1.1.0.0 and Jakarta EE 9.1.
+
+### Tags:
+  - weblogic
+  - java
+
+### Recipe Source
+
+[weblogic-15.1.1.yaml](https://github.com/oracle/rewrite-recipes/blob/main/rewrite-weblogic/src/main/resources/META-INF/rewrite/weblogic-15.1.1.yaml)
+
+### Recipe List:
+  - com.oracle.weblogic.rewrite.UpdateBuildToWebLogic1511
+  - [com.oracle.weblogic.rewrite.CheckAndCommentOutDeprecations1412](https://github.com/oracle/rewrite-recipes/blob/main/rewrite-weblogic/src/main/resources/META-INF/rewrite/weblogic-deprecations.yaml)
+
+### Usage
+
+This recipe will update the WebLogic version to 15.1.1 for the Maven build, migrate `javax` packages to `jakarta`, and comment out deprecated and removed APIs.
+
+#### Maven POM
+
+1. Add the following to your Maven POM file.
+```
+<plugin>
+    <groupId>org.openrewrite.maven</groupId>
+    <artifactId>rewrite-maven-plugin</artifactId>
+    <version>5.43.0</version>
+    <configuration>
+        <exportDatatables>true</exportDatatables>
+        <activeRecipes>
+            <recipe>com.oracle.weblogic.rewrite.UpgradeTo1511</recipe>
+        </activeRecipes>
+    </configuration>
+    <dependencies>
+        <dependency>
+            <groupId>com.oracle.weblogic.rewrite</groupId>
+            <artifactId>rewrite-weblogic</artifactId>
+            <version>0.4.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.openrewrite.recipe</groupId>
+            <artifactId>rewrite-migrate-java</artifactId>
+            <version>2.28.0</version>
+        </dependency>
+    </dependencies>
+</plugin>
+...
+```
+2. Run `mvn rewrite:run` to run the recipe.
+
+#### Gradle
+
+1. Add the following to your `build.gradle` file:
+
+```
+plugins {
+    id("org.openrewrite.rewrite") version("7.1.4")
+}
+
+rewrite {
+    activeRecipe("com.oracle.weblogic.rewrite.UpgradeTo1511")
+    setExportDatatables(true)
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    rewrite("org.openrewrite.recipe:rewrite-migrate-java:3.2.0")

```

Comment:
Is missing to add a line for `rewrite("com.oracle.weblogic.rewrite:rewrite-weblogic:0.4.2")`